src/components/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Display from './Display.jsx';
+import './App.scss';
+import ButtonPanel from './ButtonPanel.jsx';
+
+function App() {
```

Comment:
We should use `class function here` so that we can implement `render function` as mentioned in the instructions.
(Let try to have good practice of following instructions, the way you have it now will still work for now 😉, but it won't by the time you're working with state and redux store.)
![image](https://user-images.githubusercontent.com/31897434/72196001-b858b400-3415-11ea-80cd-6627d1d3305e.png)
class function example  👉  `class App extends React.Component {`

Reply:
> (Let try to have good practice of following instructions, the way you have it now will still work for now 😉, but it won't by the time you're working with state and redux store.)

I fear, that I won't accept this argument because `functional component` is not bad practice when compared to `class component`. I used `useState` react hook for state management of this project and it is perfectly fine and readable without binding hell.

Reply:
> > (Let try to have good practice of following instructions, the way you have it now will still work for now 😉, but it won't by the time you're working with state and redux store.)
> 
> I fear, that I won't accept this argument because `functional component` is not bad practice when compared to `class component`. I used `useState` react hook for state management of this project and it is perfectly fine and readable without binding hell.

Well, with `arrow function` you can write your `state` without  `constructor` and `binding`, so; it doesn't have to be because of binding hell......😉

**Don't get me wrong, it is cool to use `hooks`, but not necessarily needed in this context.

Reply:
> **Don't get me wrong, it is cool to use hooks, but not necessarily needed in this context.

Sorry, If you offended with my arguments. I already completed the whole project with react hooks I didn't do it for coolness but for its intuitiveness and to learn. Then I find context here that why I did it. I agree with function components are not hooks but changed upcoming milestones that is the reason why I can't change it now. 

The requirement said `render function should display ...`. I thought it is rendered function for class context and it is just returning for functional context. And They never said not to use hooks,

Reply:
Alright @imhta.
send in another CRR.

Reply:
Functional can go, it's ok